### ISSUE TITLE
Add default group with auto join in server config

### ISSUE THREAD

**Issue by ja2ui0** [henkelmax/simple-voice-chat#556]
**Is your feature request related to a problem? Please describe.**
When I want to chat with my team, the first person on the server has to set up a group, invite the player(s), and they have to join it. It's very complicated to use for teamchat this way, and it would be great to use this plugin for that purpose as all the functionality otherwise is already mature.

**Describe the solution you'd like**
When the server starts, I'd like to have a default group (configured in server settings) and anyone who joins the server is automatically a part of it (configurable). All anyone has to do is either PTT or voice-activated chat with no other interaction. The player could also leave the default group, if they don't want to be bothered, and create / join a different one if they want.

**Describe alternatives you've considered**
I've considered standing my mumble server back up, but integration with simple voice chat would be the ideal solution.

**Comment by henkelmax** [comment]
You can create an addon with the API that does that.
https://modrepo.de/minecraft/voicechat/api/overview

You could also request the feature here:
https://github.com/henkelmax/enhanced-groups

**Comment by ja2ui0** [comment]
Thank you for having a second look at this. I approached a couple of devs on Reddit who are looking at it right now, but having it "mainline" from the primary dev is always a good thing. 

I went to request the feature in enhanced groups, but the `issue` button prompts only for a bug report and this is hardly a bug. Anyway I reviewed the feature set in enhanced groups and it looks basically like it does what i'm asking already - particularly if the range could be set to -1 for infinite and players can opt out of it.
